src/AM.Condo/Targets/Prepare/Go.targets

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+  <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <Target Name="GetGoInfo" Returns="@(GoProjectPath)">
+      <ItemGroup>
+        <GoProjectPath Include="$(RepositoryRoot)" Condition=" Exists('$(RepositoryRoot)go.mod') " />
+        <GoProjectPath Include="$(RepositoryRoot)" Condition=" Exists('$(RepositoryRoot)go.condo') " />
+        <GoTests Include=" Exists('$(RepositoryRoot)*_test.go') " />
+        <GoTests Include=" Exists('$(RepositoryRoot)**$(slash)*_test.go') " />
+      </ItemGroup>
+
+      <PropertyGroup>
+        <GoEnabled Condition=" '$(SKIP_GO)' != '' ">false</GoEnabled>
```

Comment:
```suggestion
        <GoEnabled Condition=" '$(SKIP_GO)' != '' ">$(SKIP_GO)</GoEnabled>
```

Reply:
it would have to be the inverse of SKIP_GO since if set to true we would be setting GoEnabled to true.

```
<GoEnabled Condition=" '$(SKIP_GO)' != '' AND '$(SKIP_GO.ToLower())' == ''true" >false</GoEnabled>
```